example/create_dependencies.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
```

Comment:
This won't work on a system that only has 'python3' but I'm not sure what we do about that.
Maybe just add a README (which we should have in the examples directory anyway) and mention this limitation.

Reply:
Indeed. I've added it to the README as known issue.